plugin-maven/src/main/java/com/diffplug/spotless/maven/generic/Indent.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2016 DiffPlug
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.diffplug.spotless.maven.generic;
+
+import org.apache.maven.plugins.annotations.Parameter;
+
+import com.diffplug.spotless.FormatterStep;
+import com.diffplug.spotless.generic.IndentStep;
+import com.diffplug.spotless.maven.FormatterStepConfig;
+import com.diffplug.spotless.maven.FormatterStepFactory;
+
+public class Indent implements FormatterStepFactory {
+
+	private static final int DEFAULT_NUM_SPACES_PER_TAB = 4;
+
```

Comment:
This constant should either be defined in IndentStep, and accessed here as a `public static int defaultNumSpacesPerTab()`, or there shouldn't be a default at all.  Otherwise the `gradle` and `maven` will eventually get out of sync.  Mostly I think there shouldn't be a default at all, but if the gradle plugin already has one then good on you for matching behavior :)

Reply:
fixed: moved default value into `public static int defaultNumSpacesPerTab()` and refactored maven and gradle plugin.